ow_lander/action/ARM_MOVE_JOINT.action

```diff
@@ -0,0 +1,10 @@
+# goal definition
```

Comment:
We aren't adopting OWLAT's all-caps naming convention, right? The command unification document says the unified command action name should be "ArmMoveJoint", so I think this file should be named "ArmMoveJoint.action".